packages/connect-webextension/src/popup.ts

```diff
@@ -45,7 +45,9 @@ export class PopupManager extends EventEmitter {
         this.origin = getOrigin(settings.popupSrc);
         this.logger = logger;
         this.handshakePromise = createDeferred();
-        this.channel = new ServiceWorkerWindowChannel<PopupEventMessage>({
+        this.channel = new ServiceWorkerWindowChannel<{
+            type: typeof POPUP.CORE_LOADED | typeof POPUP.CLOSED;
```

Comment:
Previously it seemed that `POPUP.CLOSED` is a `CoreRequestMessage` (going inside connect core) but according to @karliatto it's more complex than that, so for [this commit](https://github.com/trezor/trezor-suite/commit/f18aac7243d63a33ca4c0f81b47346600bc56ec7) I fixed™ it by defining both of message types sent by this channel explicitly.

Reply:
It looks like it works. Makes sense to me.